api/spaces/export_test.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package spaces
+
+import (
+	"github.com/juju/juju/api/base"
+)
+
+func NewStateFromCaller(caller base.FacadeCaller) *API {
```

Comment:
We want to move away from naming these things as variations on "State" - `NewAPIFromCaller`.